frontend/www/js/omegaup/components/problem/creator/Header.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <b-row class="mb-3">
+    <b-col class="d-flex align-items-center">
+      <span class="mr-2">{{ T.problemCreatorName }}:</span>
```

Comment:
```suggestion
      <span class="mr-2">{{ T.problemCreatorName }}</span>
```

se puede meter el `:` a la cadena de traducción? en general las cadenas de traducción no deben de tener nada de texto alrededor, porque no nos gusta concatenar cadenas de traducción con otras cosas: hay lenguajes que usan otros símbolos (o no usan ningún símbolo!) para denotar esto.